datacapturing/src/main/java/de/cyface/datacapturing/DataCapturingService.java

```diff
@@ -1067,16 +1102,28 @@ public void handleMessage(final @NonNull Message msg) {
         }
 
         /**
-         * Adds a new listener interested in events from the background service.
+         * Adds a new {@link DataCapturingListener} interested in events from the
+         * {@link DataCapturingBackgroundService}.
          * <p>
          * All listeners are automatically removed when the {@link DataCapturingService} is killed.
          *
          * @param listener A listener that is notified of important events during data capturing.
-         * @throws IllegalStateException when the listener could not be added or was already added because it's
-         *             essential that the registration is successful or else the UI does not receive capturing events
+         * @return true if this collection changed as a result of the call
```

Comment:
Here surrounding the `true` with <code> Tags would be appropriate since it is a keyword.